compiler/noirc_evaluator/src/ssa/ir/printer.rs

```diff
@@ -140,6 +140,21 @@ fn value(dfg: &DataFlowGraph, id: ValueId) -> String {
     }
 }
 
+fn number(number: FieldElement, typ: &NumericType) -> String {
```

Comment:
I think this function could use some docstrings and some basic unit tests to serve as examples.